include/private/prival.h

```diff
@@ -0,0 +1,42 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/*
+ * Copyright 2018-2022 Joel E. Anderson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __STUMPLESS_PRIVAL_STRING
+#  define __STUMPLESS_PRIVAL_STRING
+
+/**
+ * Function for converting prival from int to string
+ *
+ * **Thread Safety: MT-Safe**
+ * This function is thread safe.
+ *
+ * **Async Signal Safety: AS-Safe**
+ * This function is safe to call from signal handlers.
+ *
+ * **Async Cancel Safety: AC-Safe**
+ * This function is safe to call from threads that may be asynchronously
+ * cancelled.
+ *
+ * @param prival int to get the string from.
+ *
+ * @return The string representation of the given prival.
+ */
+int
+stumpless_get_prival_string( int prival );
```

Comment:
Is this function used anywhere? It seems like it is duplicative of the `stumpless/prival.h` function. This header may not be needed at all.

Reply:
The redundant header has been removed. It was indeed not needed.